charts/topolvm/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+type: application
+
+home: https://github.com/topolvm/topolvm
+name: topolvm
+description: Topolvm
+version: 2.0.0
```

Comment:
I decided to raise the major version because it involves destructive changes when migrating from [topolvm/helm](https://github.com/topolvm/helm)

Reply:
Nice!